subsys/mpsl/cx/generic_3pin/mpsl_cx_generic_3pin.c

```diff
@@ -285,17 +285,17 @@ static int mpsl_cx_init(const struct device *dev)
 #if DT_NODE_HAS_PROP(CX_NODE, req_gpios)
 	uint8_t req_pin = NRF_DT_GPIOS_TO_PSEL(CX_NODE, req_gpios);
 
-	soc_secure_gpio_pin_mcu_select(req_pin, NRF_GPIO_PIN_MCUSEL_NETWORK);
+	soc_secure_gpio_pin_mcu_select(req_pin, NRF_GPIO_PIN_MCU_NETWORK);
 #endif
 #if DT_NODE_HAS_PROP(CX_NODE, pri_dir_gpios)
 	uint8_t pri_dir_pin = NRF_DT_GPIOS_TO_PSEL(CX_NODE, pri_dir_gpios);
 
-	soc_secure_gpio_pin_mcu_select(pri_dir_pin, NRF_GPIO_PIN_MCUSEL_NETWORK);
+	soc_secure_gpio_pin_mcu_select(pri_dir_pin, NRF_GPIO_PIN_MCU_NETWORK);
 #endif
 #if DT_NODE_HAS_PROP(CX_NODE, grant_gpios)
 	uint8_t grant_pin = NRF_DT_GPIOS_TO_PSEL(CX_NODE, grant_gpios);
 
-	soc_secure_gpio_pin_mcu_select(grant_pin, NRF_GPIO_PIN_MCUSEL_NETWORK);
+	soc_secure_gpio_pin_mcu_select(grant_pin, NRF_GPIO_PIN_MCU_NETWORK);
```

Comment:
`soc_secure_gpio_pin_mcu_select` in sdk-zephyr should be updated as well.

```
#if defined(GPIO_PIN_CNF_MCUSEL_Msk)
static inline void soc_secure_gpio_pin_mcu_select(uint32_t pin_number,
						  nrf_gpio_pin_mcusel_t mcu)
{
	nrf_gpio_pin_mcu_select(pin_number, mcu);
}
#endif /* defined(GPIO_PIN_CNF_MCUSEL_Msk) */
```

`#if defined(GPIO_PIN_CNF_MCUSEL_Msk)` and `nrf_gpio_pin_mcusel_t`